modules/dataflow_bucket/outputs.tf

```
@@ -0,0 +1,9 @@
+output "bucket_name" {
```

This name is redundant.

```suggestion
output "name" {
```

---

Fixed in https://github.com/terraform-google-modules/terraform-google-dataflow/pull/1/commits/b42d51328a83653787d5973ab4ea414abd0a5bb1